inputs_demand.md

```diff
@@ -0,0 +1,172 @@
+
+# Table of Contents
+
+- [Energy](#energy)
+- [Water](#water)
+- [Food, Feed, and Forestry](#food--feed--forestry)
+
+## External Inputs
+
+### Energy
+
+#### Description
+
+| Name | Description | Type | Source | Resolution | Unit |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| Historical demand for energy | Demand for energy in the historical period; used for initialization/calibration of GCAM | External data | IEA | Specified by demand, fuel, country, and year |  ktoe and GWh |
+| Price elasticity of demand | Elasticity determining how demand responds to changes in price | Assumption | | Specified by demand |  unitless |
+| Value of time in transit multiplier | Factor multiplied by the wage rate to determine the value of time in transit, used in the transportation model | Assumption | | Specified by demand |  unitless |
+| Cost | Cost of production | Assumption | | Specified by technology and year |  1975$/kg or 1975$/GJ |
+| Default input-output coefficients | Default amount of input required per unit of output produced; can be overwritten by region-specific information derived from historical data | Assumption | | Specified by technology and year |  Various (e.g., GJ per kg, GJ per GJ) |
+| Default efficiencies | Default amount of output produced per unit of input; can be overwritten by region-specific information derived from historical data | Assumption | | Specified by technology and year |  Various (e.g., GJ per kg, GJ per GJ) |
+| CO<sub>2</sub> capture rates | Fraction of CO2 captured in CCS technologies. | Assumption |  | Specified by technology and year | unitless |
+| Retirement rules | For vintaged technologies, GCAM requires the user to specify the lifetime, and the parameters required for phased and profit-based shutdown. | Assumption |  | Specified by technology and year | Years (for lifetime), unitless for others |
+| Logit exponents | GCAM requires the user to specify the logit exponents that determine the substitutability between technologies. | Assumption |  | Specified by sector and subsector | N/A |
+| Share weight interpolation rules | These rules dictate how share weights (GCAM's calibration parameter) are specified in future years. | Assumption |  | Specified by sector and subsector | N/A |
+| Fuel preference elasticity | Elasticity dictating how share weights change with GDP per capita | Assumption | Specified by technology and year | unitless |
```

Comment:
Fuel preference elasticity - there should be an additional column between "Assumption" and "Specified by technology and year". 
Assumption |  | Specified by technology and year